menas/ui/components/schema/schemaMain.controller.js

```diff
@@ -222,10 +227,21 @@ sap.ui.controller("components.schema.schemaMain", {
 			// nav back to info
 			this.byId("schemaIconTabBar").setSelectedKey("info");
 		}
+	} ,
+
+  validateSchemaFileUplaod : function () {
+    this.resetNewSchemaValueState();
+    let isOk = true;
+    let oSchemaFileUpload = this.byId("fileUploader").getValue();
+    if (oSchemaFileUpload === "" ) {
```

Comment:
`oSchemaFileUpload` is not used anywhere else then the condition. Why using a local varibale and not just a direct  function call?

Reply:
I feel this cleaner that the other.

Reply:
Fair enough.

Reply:
Thanks.